acceptance/tests/reports/submission.rb

```diff
@@ -71,7 +71,8 @@ def query_last_report_time_on(agent)
     agents.each do |agent|
       on(agent, puppet('agent', "-t --server #{master}"))
 
-      on master, "grep -q #{agent} #{testdir}/*/*"
+      hostname = fact_on(agent, 'hostname')
```

Comment:
I thought beaker had a method for getting the host's idea of its name (perhaps using facter)?